### ISSUE TITLE
tr failing tr_tests::test_tr_s

### ISSUE THREAD

**Issue by jgarzik** [rustcoreutils/posixutils-rs#129]
Both MacOS and Linux fail in the same way, on the same test:
```
failures:

---- tr_tests::test_tr_s stdout ----
thread 'tr_tests::test_tr_s' panicked at plib/src/testing.rs:56:5:
assertion `left == right` failed
  left: ""
 right: "The\nbig\nblack\nfox\njumped\nover\nthe\nfence\n"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    tr_tests::test_tr_s

test result: FAILED. 290 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 2.46s

error: test failed, to rerun pass `-p posixutils-text --test integration
```

**Comment by Wandalen** [comment]
My bad. Fixing

**Comment by Wandalen** [comment]
https://github.com/rustcoreutils/posixutils-rs/pull/130

Here it is

**Comment by Wandalen** [comment]
Inaccurate merge 🤷‍♂️

**Comment by jgarzik** [comment]
Fixed by #130
